src/framework/vst/internal/vstpluginprovider.cpp

```
@@ -80,5 +79,8 @@ PluginControllerPtr VstPluginProvider::controller() const
 
 PluginMidiMappingPtr VstPluginProvider::midiMapping() const
```

Cppcheck (2.20.0)

[style] The function 'midiMapping' is never used.

(unusedFunction)